impl/src/test/java/org/ehcache/internal/store/heap/BaseOnHeapStoreTest.java

```diff
@@ -624,14 +626,20 @@ public String apply(String mappedKey) {
 
     assertThat(newValue.value(), equalTo("value2"));
     assertThat(store.get("key").value(), equalTo("value2"));
-    verify(listener).onExpiration(argThat(new ArgumentMatcher<Cache.Entry<String, String>>() {
+    verify(listener).onExpiration(argThat(new ArgumentMatcher<String>() {
```

Comment:
First `argThat(...)` can be replaced by `eq("key")` here and in other tests, second one should be extracted to an internal method to remove the code duplication across different tests.